lmdeploy/pytorch/models/qwen.py

```diff
@@ -11,27 +11,51 @@
                           rowwise_parallelize_linear_fn, try_to_local)
 from ..kernels import apply_rotary_pos_emb, fill_kv_cache, paged_attention_fwd
 
+def _attention_partition_fn(mod_name: str, mod: nn.Module,
```

Comment:
@jjjjohnson hi, could you fix the lint error?   THX

Reply:
Done